types/index.d.ts

```diff
@@ -1140,6 +1140,12 @@ declare namespace RamdaAdjunct {
         argsPass<T>(combiningPredicate: (fn: (a: T) => boolean) => (list: T[]) => boolean, predicates: Pred[]): Pred;
         argsPass<T>(combiningPredicate: (fn: (a: T) => boolean) => (list: T[]) => boolean): (predicates: Pred[]) => Pred;
 
+        /**
+         * Takes a function as an argument and a number of additional arguments, and returns a function that just calls the original function given.
+         * If any of the arguments to the function that it returns are null or undefined, then the original default argument is used instead.
+         */
+         fNull(fn: Function, defaultArgs: [any]): Function;
```

Comment:
```suggestion
         fnull(fn: Function, defaults: any[]): Function;
         fnull(fn: Function): (defaults: any[]) => Function;  
```

Reply:
We're missing the tests for these typings: please have a look at model PR which shows what needs to be done: https://github.com/char0n/ramda-adjunct/pull/1287/files